course_catalog/etl/enrollments.py

```diff
@@ -0,0 +1,78 @@
+"""Course run enrollments"""
+import pyathena
+import pytz
+from pyathena.cursor import DictCursor
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from course_catalog.models import LearningResourceRun, Course, Enrollment
+
+
+def athena_cursor():
+    """Returns authorized pyathena cursor"""
+    return pyathena.connect(
+        aws_access_key_id=settings.ATHENA_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.ATHENA_SECRET_ACCESS_KEY,
+        work_group=settings.ATHENA_WORK_GROUP,
+        cursor_class=DictCursor,
+        region_name=settings.ATHENA_REGION_NAME,
+    ).cursor()
+
+
+def update_enrollments_for_user(user):
+    """
+    Updates enrollment data for a singe user
+
+    Args:
+        user (User): user to fetch enrollments for
+    """
+
+    if not (
+        settings.ATHENA_ACCESS_KEY_ID
+        and settings.ATHENA_SECRET_ACCESS_KEY
+        and settings.ATHENA_WORK_GROUP
+        and settings.ATHENA_REGION_NAME
+    ):
+        return
+
+    query = """
+        SELECT 
+            enrollment_course_id,
+            enrollment_created
+        FROM ol_warehouse.mitx_enrollments 
+        WHERE email = '{email}'
+        """.format(
+        email=user.email
+    )
+
+    cursor = athena_cursor()
```

Comment:
I think it would make sense to pass the cursor as a parameter to this function so that it is easier to swap to Trino from Athena once we have that deployed. It would also simplify mocking the cursor object for tests.

Reply:
Once we remove the check for credentials above, swapping the cursor will just involve changing athena_cursor()
 to trino_curosr() here. Passing the cursor to this functions works too though